doc/examples/transform/plot_transform_types.py

```diff
@@ -0,0 +1,142 @@
+"""
+===================================
+Types of homographies
+===================================
+
+`Homographies <https://en.wikipedia.org/wiki/Homography>`_
+are transformations of a Euclidean space that preserve the alignment of points.
+Specific cases of homographies correspond to the conservation of more properties,
+such as parallelism (affine transformation), shape (similar transformation) or distances (Euclidean transformation).
+
+Homographies on a 2D Euclidean space (i.e., for 2D grayscale or multichannel
+images) are defined by a 3x3 matrix. All types of homographies can be defined
+by passing either the transformation matrix, or the parameters of the simpler
+transformations (rotation, scaling, ...) which compose the full transformation.
+
+The different types of homographies available in scikit-image are
+shown here, by increasing order of complexity. While we focus here on the
+mathematical properties of transformations, tutorial :ref:`sphx_glr_auto_examples_transform_plot_geometric.py`
+explains how to use such transformations for various tasks such as image warping or parameter estimation.
+"""
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage import transform as tf
+from skimage import img_as_float
+
+######################################################################
+# Euclidean (rigid) transformation
+# =================================
+#
+# A `Euclidean transformation <https://en.wikipedia.org/wiki/Rigid_transformation>`_,
+# also called rigid transformation, preserves the Euclidean distance between
+# pairs of points. It can be described as a rotation about the origin
+# followed by a translation.
+
+tform = tf.EuclideanTransform(
+    rotation=np.pi / 12.,
+    translation = (100, -20)
+    )
+print(tform.params)
+
+######################################################################
+# Now let's apply this transformation to an image. Because we are trying
+# to reconstruct the *image* after transformation, it is not useful to see
+# where a *coordinate* from the input image ends up in the output, which is
+# what the transform gives us. Instead, for every pixel (coordinate) in the
+# output image, we want to figure out where in the input image it comes from.
+# Therefore, we need to use the inverse of ``tform``, rather than ``tform``
+# directly.
+
+img = img_as_float(data.chelsea())
+tf_img = tf.warp(img, tform.inverse)
+_ = plt.imshow(tf_img)
+plt.title('Euclidean transformation')
+
+
+######################################################################
+# For a rotation around the center of the image, one can 
+# compose a translation to change the origin, a rotation, and finally
+# the inverse of the first translation.
+# in degrees, while it is given in radians for the different kinds of
+# affine transformations):
```

Comment:
```suggestion
```

Reply:
ok I implemented this directly instead of using the suggestion, so I'll resolve the conversation